packages/react-core/src/helpers/Popper/thirdparty/react-popper/usePopper.ts

```diff
@@ -20,7 +20,7 @@ const fromEntries = (entries: [string, any][]) =>
 /**
  * Small wrapper around `useLayoutEffect` to get rid of the warning on SSR envs
  */
-const useIsomorphicLayoutEffect =
+export const useIsomorphicLayoutEffect =
   canUseDOM
     ? React.useLayoutEffect
     : React.useEffect;
```

Comment:
This is a strange place for this function to live. Can you move it to a new `src/helpers/useIsomorphicLayout.ts` file that gets reexported from `src/helpers/index.ts` instead?